server/data_processors/BarChartDataProcessor/BarChartDataProcessor.test.js

```diff
@@ -50,14 +50,14 @@ describe(BarChartDataProcessor, () => {
         },
       }
       const expectedDataMap = {
-        'Madrid-Bar-2': 0,
-        'Madrid-Foo-3': 4,
+        'Madrid-Bar-MA-2': 0,
```

Comment:
Is this to fix a duplicate key error?

Reply:
This is to have the site code available to display on the UI. If there are many sites the width of the axis is small and the sites names are quite long, this makes it so we can display the short code and when we hover over the short code we get the full name.